src/phases/encounter-phase.ts

```diff
@@ -439,13 +439,13 @@ export class EncounterPhase extends BattlePhase {
     if (![BattleType.TRAINER, BattleType.MYSTERY_ENCOUNTER].includes(this.scene.currentBattle.battleType)) {
       enemyField.map(p => this.scene.pushConditionalPhase(new PostSummonPhase(this.scene, p.getBattlerIndex()), () => {
         // if there is not a player party, we can't continue
-        if (!this.scene.getParty()?.length) {
+        if (!this.scene.getPlayerParty()?.length) {
```

Comment:
NIT: Player-party will never be null/undefined
```suggestion
        if (!this.scene.getPlayerParty().length) {
```